firebase-firestore/src/main/java/com/google/firebase/firestore/local/SQLiteSchema.java

```diff
@@ -384,6 +384,7 @@ private void createFieldIndex() {
                   + "sequence_number INTEGER, " // Specifies the order of updates
                   + "read_time_seconds INTEGER, " // Read time of last processed document
                   + "read_time_nanos INTEGER, "
+                  + "document_key TEXT, " // Key of the last processed document
```

Comment:
nit: add corresponding document_key test in SQLiteSchemaTest

Reply:
I am not sure I follow here. What test are you referring to? The test I spotted only validated the table names.

Reply:
Nevermind, it was a test in my imagination.